internal/pkg/externalplugins/label/label.go

```diff
@@ -59,15 +59,15 @@ func HelpProvider(epa *externalplugins.ConfigAgent) func(
 
 		pluginHelp := &pluginhelp.PluginHelp{
 			Description: "The label plugin provides commands that add or remove certain types of labels. " +
-				"Labels of the following types can be manipulated: 'status/*'," +
-				" 'sig/*', and 'type/*'. More labels can be configured to be used via the /label command.",
+				"For example, the labels of the following types: 'status/*'," +
+				" 'sig/*', and 'type/*'.",
 			Config: labelConfig,
 		}
 		pluginHelp.AddCommand(pluginhelp.Command{
-			Usage:       "/[remove-](status|sig|kind|label) <target>",
-			Description: "Applies or removes a label from one of the recognized types of labels.",
+			Usage:       "/[remove-](status|sig|type|label|component) <target>",
+			Description: "Add or remove a label of the given type.",
 			Featured:    false,
-			WhoCanUse:   "Anyone can trigger this command on a PR.",
+			WhoCanUse:   "Everyone can trigger this command.",
```

Comment:
Please use `anyone` because we use `anyone` in label.yaml.

Reply:
You may want to take a look: https://zhidao.baidu.com/question/415859367.html

Reply:
Got it.
Could you also modify all anyone in the labels.yaml?

Reply:
done